src/scripts/glue-etl.py

```diff
@@ -136,16 +153,30 @@ def sum_col(df, col):
ataframe.V216,spark_dataframe.V217,spark_dataframe.V218,spark_dataframe.V219,spark_dataframe.V220,spark_dataframe.V221,spark_dataframe.V222,spark_dataframe.V223,spark_dataframe.V224,spark_dataframe.V225,spark_dataframe.V226,spark_dataframe.V227,spark_dataframe.V228,spark_dataframe.V229,spark_dataframe.V230,spark_dataframe.V231,spark_dataframe.V232,spark_dataframe.V233,spark_dataframe.V234,spark_dataframe.V235,spark_dataframe.V236,spark_dataframe.V237,spark_dataframe.V238,spark_dataframe.V239,spark_dataframe.V240,spark_dataframe.V241,spark_dataframe.V242,spark_dataframe.V243,spark_dataframe.V244,spark_dataframe.V245,spark_dataframe.V246,spark_dataframe.V247,spark_dataframe.V248,spark_dataframe.V249,spark_dataframe.V250,spark_dataframe.V251,spark_dataframe.V252,spark_dataframe.V253,spark_dataframe.V254,spark_dataframe.V255,spark_dataframe.V256,spark_dataframe.V257,spark_dataframe.V258,spark_dataframe.V259,spark_dataframe.V260,spark_dataframe.V261,spark_dataframe.V262,spark_dataframe.V263,spark_dataframe.V264,spark_dataframe.V265,spark_dataframe.V266,spark_dataframe.V267,spark_dataframe.V268,spark_dataframe.V269,spark_dataframe.V270,spark_dataframe.V271,spark_dataframe.V272,spark_dataframe.V273,spark_dataframe.V274,spark_dataframe.V275,spark_dataframe.V276,spark_dataframe.V277,spark_dataframe.V278,spark_dataframe.V279,spark_dataframe.V280,spark_dataframe.V281,spark_dataframe.V282,spark_dataframe.V283,spark_dataframe.V284,spark_dataframe.V285,spark_dataframe.V286,spark_dataframe.V287,spark_dataframe.V288,spark_dataframe.V289,spark_dataframe.V290,spark_dataframe.V291,spark_dataframe.V292,spark_dataframe.V293,spark_dataframe.V294,spark_dataframe.V295,spark_dataframe.V296,spark_dataframe.V297,spark_dataframe.V298,spark_dataframe.V299,spark_dataframe.V300,spark_dataframe.V301,spark_dataframe.V302,spark_dataframe.V303,spark_dataframe.V304,spark_dataframe.V305,spark_dataframe.V306,spark_dataframe.V307,spark_dataframe.V308,spark_dataframe.V309,spark_dataframe.V310,spark_dataframe.V311,spark_dataframe.V312,spark_dataframe.V313,spark_dataframe.V314,spark_dataframe.V315,spark_dataframe.V316,spark_dataframe.V317,spark_dataframe.V318,spark_dataframe.V319,spark_dataframe.V320,spark_dataframe.V321,spark_dataframe.V322,spark_dataframe.V323,spark_dataframe.V324,spark_dataframe.V325,spark_dataframe.V326,spark_dataframe.V327,spark_dataframe.V328,spark_dataframe.V329,spark_dataframe.V330,spark_dataframe.V331,spark_dataframe.V332,spark_dataframe.V333,spark_dataframe.V334,spark_dataframe.V335,spark_dataframe.V336,spark_dataframe.V337,spark_dataframe.V338,spark_dataframe.V339,spark_dataframe.M1_F,spark_dataframe.M1_T,spark_dataframe.M2_F,spark_dataframe.M2_T,spark_dataframe.M3_F,spark_dataframe.M3_T,spark_dataframe.M4_M0,spark_dataframe.M4_M1,spark_dataframe.M4_M2,spark_dataframe.M5_F,spark_dataframe.M5_T,spark_dataframe.M6_F,spark_dataframe.M6_T,spark_dataframe.M7_F,spark_dataframe.M7_T,spark_dataframe.M8_F,spark_dataframe.M8_T,spark_dataframe.M9_F,spark_dataframe.M9_T))
```

Comment:
create the props as json string not hard-code the name of columns,

for example,

```python
spark_dataframe = spark_dataframe.select(spark_dataframe.TransactionID, to_json(struct(list(filter(lambda x: (x != TRANSACTION_ID), spark_dataframe.schema.names)))).alias('props_values:String’))
```